examples/swapi/swapi-loaders.js

```diff
@@ -4,29 +4,29 @@
  * !!! THIS FILE IS AUTO-GENERATED. CHANGES MAY BE OVERWRITTEN !!!
  */
 
-import util from 'util';
-import _ from 'lodash';
-import invariant from 'assert';
-import DataLoader from 'dataloader';
+import util from "util";
```

Comment:
do you know why the file formatting changed on this? probably fine but just curious if there's a new version of prettier or eslint or something causing all this (e.g. what's causing `'` -> `"`?)

Reply:
not quite sure about this... prettier versions are not updated. how could we test if this will be an issue